.github/workflows/api-release.yml

```
@@ -0,0 +1,38 @@
+    env:
+      SIGN_KEY_PASS: ${{ secrets.gpg_passphrase }}
+      CENTRAL_USERNAME: ${{ secrets.sonatype_username }}
+      CENTRAL_PASSWORD: ${{ secrets.sonatype_password }}
+  - name: Create GitHub Release
+    id: create_release
+    uses: softprops/action-gh-release@v1
+    with:
+      tag_name: ${{ github.event.inputs.release-tag }}
+      draft: false
+      prerelease: false
+      generate_release_notes: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)